pkg/morph/client/netmap/netmap.go

```diff
@@ -107,6 +145,103 @@ func (c *Client) EpochSnapshot(args EpochSnapshotArgs) (*EpochSnapshotValues, er
 	}, nil
 }
 
+func (c *Client) Candidates(_ GetNetMapCandidatesArgs) (*GetNetMapCandidatesValues, error) {
+	prms, err := c.client.TestInvoke(
+		c.netMapCandidatesMethod,
+	)
+	if err != nil {
+		return nil, fmt.Errorf("could not perform test invocation (%s): %w", c.netMapCandidatesMethod, err)
+	}
+
+	candVals, err := peersWithStateFromStackItems(prms, c.netMapCandidatesMethod)
+	if err != nil {
+		return nil, fmt.Errorf("could not parse contract response: %w", err)
+	}
+
+	return candVals, nil
+}
+
+func peersWithStateFromStackItems(stack []stackitem.Item, method string) (*GetNetMapCandidatesValues, error) {
+	if ln := len(stack); ln != 1 {
+		return nil, fmt.Errorf("unexpected stack intem count (%s): %d", method, ln)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unexpected stack item count (%s): %d", method, ln)
```
and re-check if it caused by copy-paste.

Reply:
fixed and checked